bfvmm/src/exit_handler/src/exit_handler_intel_x64.cpp

```diff
@@ -934,3 +1028,132 @@ exit_handler_intel_x64::exit_reason_to_str(uint64_t exit_reason)
             return "unknown";
     };
 }
+
+void
+exit_handler_intel_x64::handle_vmcall_versions(vmcall_registers_t &regs)
+{
+    switch (regs.r02)
+    {
+        case 0:
```

Comment:
I could add an enum to this interface if you would like, but they are intended to be arbitrary as the user can add to this as they wish